app/controllers/measurements_controller.rb

```diff
@@ -56,6 +56,40 @@ def new_meteorological
     end
   end
 
+  def new_wind_turbine_speed
+    accepted = {}
+    accepted[:rpm] = params[:rpm]
+    @wind_turbine_speed_measurement = WindTurbineSpeedMeasurement.new(accepted)
+    attempt = @wind_turbine_speed_measurement.save
+    if attempt
+      create_notifications(WindTurbineSpeedMeasurement)
+      render html: "WindTurbineSpeedMeasurements #{accepted} saved successfully!", layout: true
+    else
+      render html: "Error saving to DB. Please check your GET URL.", layout: true
+    end
+  end
+
+  def new_wind_turbine_vibration
```

Comment:
Speed is the only field of the WindTurbineSeedMeasurement model. Could we use only one model/table called WindTurbineMeasurement and include there all the fields related to the WindTurbine?